stored_requests/backends/db_provider/mysql_dbprovider.go

```diff
@@ -75,7 +84,65 @@ func (provider *MySqlDbProvider) ConnString() string {
 		buffer.WriteString(provider.cfg.Database)
 	}
 
-	return buffer.String()
+	// TSL connection
+	if provider.cfg.ServerCa != "" && provider.cfg.ClientCert != "" && provider.cfg.ClientKey != "" {
+		if err := setupTLSConfig(provider); err != nil {
+			return "", err
+		}
+		buffer.WriteString("?tls=prebid-tls")
+	}
+
+	return buffer.String(), nil
+}
+
+func setupTLSConfig(provider *MySqlDbProvider) error {
```

Comment:
Thank you for your contribution. Changes in this PR make sense to me, I found related documentation here: 
https://pkg.go.dev/github.com/go-sql-driver/mysql#RegisterTLSConfig
https://cloud.google.com/sql/docs/mysql/samples/cloud-sql-mysql-databasesql-connect-tcp-sslcerts

I checked it works with my local MySQL and Postgres DBs. 
However I din't find how to configure my local MySQL server to support certificates.